### ISSUE TITLE
JettyHttpContainerFactory no longer being exposed.

### ISSUE THREAD

**Issue by hunterbreathat-wk** [eclipse-ee4j/jersey#4739]
👋🏻 In transitioning to version `3.0.1` of `jersey-containers-jetty-http` it was noticed that 
```kotlin
server = JettyHttpContainerFactory.createServer(baseUri, config)
```
no longer functions. This is due to that class not being exposed, which is the opposite of what the documentation suggests, found [here](https://eclipse-ee4j.github.io/jersey.github.io/documentation/3.0.1/deployment.html#deployment.http) under section 4.5.1.3. This was also not listed as a breaking change, I currently have a workaround (copy the setup from that class definition).

I'm not sure if this was intentional on release or not.

PS. The JettyHttpContainerFactory.class is included in the meta of the lib but not actually exposed outwards.

**Comment by senivam** [comment]
The issue with Jetty multi-release jar is fixed under this issue: #4655, the fix shall become available under Jersey 3.0.2 version which will be released in the nearest feature.

**Comment by hunterbreathat-wk** [comment]
Would it be possible for a SNAPSHOT to be released for `3.0.2`? If not what is the projected release timeline?
